orchagent/aclorch.cpp

```diff
@@ -939,14 +983,25 @@ bool AclTable::create()
     attr.value.booldata = true;
     table_attrs.push_back(attr);
 
-    attr.id = SAI_ACL_TABLE_ATTR_FIELD_SRC_IP;
-    attr.value.booldata = true;
-    table_attrs.push_back(attr);
+    if (type == ACL_TABLE_L3V6) {
```

Comment:
>{ [](start = 32, length = 1)

can you leave the brace in a separate line?

Reply:
done